packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-sell-button.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { getCardLabels, isValidToSell } from '@deriv/shared';
+import { Button, Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import { observer, useStore } from '@deriv/stores';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TAccumSellButton = {
+    contract_info?: TContractInfo;
+    current_stake: number | null;
+    currency?: string;
+    is_disabled: boolean;
+    is_sell_requested?: boolean;
+    onClick: (e: React.MouseEvent<HTMLElement>) => void;
+};
+const AccumulatorsSellButton = observer(
+    ({ contract_info, current_stake, currency, is_disabled, is_sell_requested, onClick }: TAccumSellButton) => {
+        const { ui } = useStore();
+        const { is_dark_mode_on } = ui;
```

Comment:
It was considered, that in dark mode it's hard to read the text as `--text-disabled` ( #999 ) is converted to #3E3E3E; I was requested to change it to #6E6E6E (`--text-less-prominent`)